beacon-chain/state/v3/setters_state.go

```diff
@@ -2,9 +2,29 @@ package v3
 
 import (
 	"github.com/pkg/errors"
+	customtypes "github.com/prysmaticlabs/prysm/beacon-chain/state/custom-types"
 	"github.com/prysmaticlabs/prysm/beacon-chain/state/stateutil"
 )
 
+// SetStateRoots for the beacon state. Updates the state roots
+// to a new value by overwriting the previous value.
+func (b *BeaconState) SetStateRoots(val *[8192][32]byte) error {
```

Comment:
This was missing in v3. I will prepare a PR against `develop` with this function.